corehq/apps/reports/tasks.py

```diff
@@ -176,20 +176,17 @@ def _store_excel_in_blobdb(report_class, file, domain):
 @task(serializer='pickle')
 def build_form_multimedia_zip(
         domain,
-        xmlns,
-        startdate,
-        enddate,
-        app_id,
         export_id,
-        zip_name,
-        download_id,
+        datespan,
         user_types,
+        download_id,
 ):
+    from corehq.apps.export.models import FormExportInstance
+    export = FormExportInstance.get(export_id)
```

Comment:
This was previously fetched inside a helper function - I just moved the call up the stack